optimizer/lib/transformers/AddBlurryImagePlaceholders.js

```diff
@@ -74,15 +74,36 @@ class AddBlurryImagePlaceholders {
    * @param {TreeAdapter} tree A parse5 treeAdapter.
    * @param {String} src The image that the bitmap is based on.
    * @return {!Promise} A promise that signifies that the img has been updated
-   * to have correct attributes to be a blurred placeholder.
+   * to have correct attributes to be a blurred placeholder along with the
+   * placeholder itself.
    * @private
    */
   addBlurryPlaceholder_(tree, src) {
     const img = tree.createElement('img');
-    img.attribs.src = src;
-    img.attribs.class = 'i-amphtml-blur';
+    img.attribs.class = 'i-amphtml-blurry-placeholder';
     img.attribs.placeholder = '';
-    return this.getDataURI_(img).then(() => {
+    img.attribs.src = src;
+    return this.getDataURI_(img).then((dataURI) => {
+      const svgOpen = '<svg xmlns="http://www.w3.org/2000/svg" '
```

Comment:
nit: for readability I'd prefer this to be a  properly formatted multi-line string literal. 

```
`<svg xmlns="http://www.w3.org/2000/svg" 
       xmlns:xlink="http://www.w3.org/1999/xlink>
    <image filter="url(#b)" x="0" y="0" ...
`
```

You can update `svg.replace(/%20/g, ' ');` to remove multiple whitespaces resulting from the formatting.